build/javascript.mk

```diff
@@ -39,6 +39,7 @@ javascript-wasm: apply-patches lua53 milagro embed-lua
 
 javascript-rn: cflags += -DARCH_JS -D'ARCH=\"JS\"' -D MAX_STRING=128000
 javascript-rn: ldflags += -s WASM=0 \
+	-s ENVIRONMENT=\"'shell'\" \
```

Comment:
I really don't have a way to test this... no project made with react-native by myself... but ok on trust and will check it soon over the decode app